string name
vigir_pluginlib_msgs/PluginDescription[] descriptions
---
